locales-pending/fix.ftl

```diff
@@ -0,0 +1,63 @@
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+fix-flow-nav-data-broker-profiles = Data broker profiles
+fix-flow-nav-high-risk-data-breaches = High risk data breaches
+fix-flow-nav-leaked-passwords = Leaked passwords
+fix-flow-nav-security-recommendations = Security recommendations
+fix-flow-data-broker-profiles-start-free-scan-headline = Find out which sites are selling your personal info
+fix-flow-data-broker-profiles-start-free-scan-content-p1 = Before you review your data breaches, would you like us to search for you on 190 data broker sites that may be selling your personal information? 
+fix-flow-data-broker-profiles-start-free-scan-content-p2 = Your first scan is free and only takes a few minutes.
+fix-flow-data-broker-profiles-start-free-scan-link-learn-more = Learn more
+fix-flow-data-broker-profiles-start-free-scan-button-start-scan = Start free scan
+fix-flow-data-broker-profiles-start-free-scan-button-skip = Skip for now
+# Variables:
+# $data_broker_sites_results_num is the number of data broker sites that have your data
+fix-flow-data-broker-profiles-view-data-broker-profiles-headline = 
+  { $data_broker_sites_results_num ->
+      [one] { $data_broker_sites_results_num } site is selling your personal information
+      *[other] { $data_broker_sites_results_num } sites are selling your personal information
+  }
+fix-flow-data-broker-profiles-view-data-broker-profiles-content = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. Remove these now to protect your privacy.
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-for-me = Remove them for me
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-manually = I’ll remove them manually
+fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = View your info on these sites
+fix-flow-data-broker-profiles-view-data-broker-profiles-view-profile = View your profile
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-view-more = View more  
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-view-less = View less  
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-headline = Here’s how to remove these profiles 
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-title = Visit each site and find removal instructions
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-content = They may be called data suppression, delete my data, do not show my data, or opt out instructions.
```

Comment:
I think this should be `opt-out instructions`, as it's the noun here 🤔

Reply:
Good catch. Revised in 6e660ee1c87da0f3c7f9648f7c9868b698bbf957